x-pack/plugins/maps/public/components/layer_panel/join_editor/resources/metrics_expression.js

```diff
@@ -70,6 +74,13 @@ export class MetricsExpression extends Component {
         return `${type} ${field}`;
       });
 
+    const useMetricDescription = metricExpressions.length > 1 ?
+      i18n.translate('xpack.maps.layerPanel.metricsExpression.useMetricsMultipleDescription', {
+        defaultMessage: 'and use metrics'
+      }) :
+      i18n.translate('xpack.maps.layerPanel.metricsExpression.useMetricsSingleDescription', {
```

Comment:
for pluralization cases we have use this instead: https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#pluralization